core-bundle/src/Entity/Altcha.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Entity;
+
+use Doctrine\ORM\Mapping\Column;
+use Doctrine\ORM\Mapping\Entity;
+use Doctrine\ORM\Mapping\GeneratedValue;
+use Doctrine\ORM\Mapping\Id;
+use Doctrine\ORM\Mapping\Index;
+use Doctrine\ORM\Mapping\Table;
+
+#[Table(name: 'altcha_challenges')]
+#[Entity]
+#[Index(columns: ['challenge'], name: 'challenge')]
+#[Index(columns: ['expires'], name: 'expires')]
+class Altcha
+{
+    #[Id]
+    #[Column(type: 'integer', options: ['unsigned' => true])]
+    #[GeneratedValue]
+    protected int $id;
+
+    #[Column(type: 'string', length: 64, nullable: false)]
+    protected string $challenge;
+
+    #[Column(type: 'datetime')]
+    protected \DateTimeInterface $created;
+
+    #[Column(type: 'datetime')]
+    protected \DateTimeInterface $expires;
```

Comment:
We don‘t actually need this field anymore, do we? We can just do `WHERE created < nowMinus24Hours`.